include/alpaka/api/host/Device.hpp

```diff
@@ -52,6 +52,11 @@ namespace alpaka::onHost
                 return m_idx != other.m_idx;
             }
 
+            void wait()
+            {
+                // Host device operations are synchronous, so no-op
```

Comment:
host operations are not synchonious. You need to itwrate over the list of streams and call wait on each stream. I can explain it next week if you like.